java-api-examples/run-non-streaming-decode-file-fire-red-asr-ctc.sh

```
@@ -2,28 +2,7 @@
-
-  make -j4
-  ls -lh lib
-  popd
-fi
-
-if [ ! -f ../sherpa-onnx/java-api/build/sherpa-onnx.jar ]; then
-  pushd ../sherpa-onnx/java-api
-  make
-  popd
-fi
+source ./setup.sh
```

Shellcheck (0.11.0)

[info] Not following: ./setup.sh was not specified as input (see shellcheck -x).

(SC1091)